src/optimizer/index_util.cpp

```diff
@@ -88,6 +88,7 @@ bool IndexUtil::CheckPredicates(
   // To concatenate/shrink ranges, we would need to be able to compare TransientValues.
   std::unordered_map<catalog::indexkeycol_oid_t, planner::IndexExpression> open_highs;  // <index, low start>
   std::unordered_map<catalog::indexkeycol_oid_t, planner::IndexExpression> open_lows;   // <index, high end>
+  int flag = 1;
```

Comment:
Can this variable be named something more descriptive?

Reply:
> Can this variable be named something more descriptive?

Yes, I have changed the name from flag to left_side and change the type.